tap_frontapp/schemas.py

```diff
@@ -6,13 +6,16 @@
 
 class IDS(object): # pylint: disable=too-few-public-methods
     TEAM_TABLE = 'team_table'
+    CONVERSATIONS = 'conversations'
 
 STATIC_SCHEMA_STREAM_IDS = [
-    IDS.TEAM_TABLE
+    IDS.TEAM_TABLE,
+    IDS.CONVERSATIONS
 ]
 
 PK_FIELDS = {
-    IDS.TEAM_TABLE: ['analytics_date', 'analytics_range', 'teammate_v']
+    IDS.TEAM_TABLE: ['analytics_date', 'analytics_range', 'teammate_v'],
+    IDS.CONVERSATIONS: 'id'
```

Comment:
Does this need to be a list? The [Singer spec](https://github.com/singer-io/getting-started/blob/bbb53c62e727b436e6952211b0761b76c0ccc41b/docs/SPEC.md#schema-message) requires primary keys to be a list in Schema messages.